### ISSUE TITLE
Custom location invalid if contains spaces

### ISSUE THREAD

**Issue by sabinrimni** [dragoonDorise/EmuDeck#769]
I'm trying to install EmuDeck 2.1.4 on Fedora KDE 38 and I'm getting a "non writable directory selected" error when choosing a custom location. In the logs I can see this error:
```
[05/25/2023] customLocation
[05/25/2023] testLocationValid "custom" "/run/media/<user>/App Data SSD/Games/EmuDeck"
[05/25/2023] stderr: Invalid: custom contains spaces
```

**Comment by Godsbane** [comment]
Yes, spaces not supported in the Emulation path.

Lot of validation would need to happen to fix it, but pull requests welcome.

**Comment by sabinrimni** [comment]
Thanks, that's good to know... I couldn't easily find anything online about it and wasn't sure if it's a bug... I fixed it by changing the mount points to remove the spaces. I have a feeling like it's best for future compatibility anyway (relatively new Linux Desktop user, still getting the hang of things).

As for the PR... as much as I have some ideas on how this could be reasonably done (_theoretically_), I recently figured out that in Linux different things (_sometimes_) require different ways to escape `space`... so I understand why this is a major pain and wouldn't really want to attempt it.
